android/src/main/java/io/rownd/android/views/RowndWebView.kt

```diff
@@ -168,6 +190,7 @@ class RowndWebViewClient(webView: RowndWebView) : WebViewClient() {
 
     @OptIn(ExperimentalMaterialApi::class)
     override fun onPageFinished(view: WebView?, url: String?) {
+        timeout = false
```

Comment:
We can likely move this to `onPageStarted` to prevent slow loading pages from triggering the no internet page.